content/errors/index.textile

```diff
@@ -0,0 +1,122 @@
+---
+title: Debugging
+meta_description: "Debugging in Ably supported apps, including troubleshooting techniques, logging options, and tools for error analysis."
+languages:
+- javascript
+---
+
+Errors can occur in various scenarios when using Ably, such as invalid inputs in requests, authentication issues, or connection problems caused by network disruptions. Proper debugging is essential for building a reliable application and troubleshooting.
+
+You can debug issues in your Ably-supported app using the following:
+
+* Set up a custom log handler to capture and manage errors in a way that suits your requirements.
+* Meta channels allow you to monitor errors that might not otherwise be visible to clients, providing additional insights into issues.
+* The Dev console in your Ably "dashboard":https://ably.com/accounts/any is a quick and easy way to inspect errors and events, especially during development or debugging.
+
+h2(#format). Error info
+
+All errors returned by Ably are standardized and use the "@ErrorInfo@":/docs/api/rest-sdk/types#error-info object:
+
+```[text]
+{
+  code: 40005,
+  statusCode: 400,
+  cause: Authentication,
+  nonfatal: false,
+  href: 'https://help.ably.io/error/40005'
+}
+```
+
+The following explains each property of an @ErrorInfo@ object:
+
+- @code@ := Ably-specific numeric code indicating the error type.
+- @statusCode@ := An HTTP status code providing broader context, such as 400 for a bad request.
+- @cause@ := A brief description of the issue.
+- @nonfatal@ := A boolean indicating whether the error is critical.
+- @Href@ := A direct link to Ably's documentation or for quick troubleshooting references.
+
+h2(#logging). Logging
+
+Most Ably SDKs allow you to customize the function that handles logging. This function is usually set in the options when configuring a client, such as the @ClientOptions@ object for Pub/Sub.
+
+Two separate properties can be set:
+
+- @logHandler@ := Provides a custom function for each line of log output.
+- @logLevel@ := The verbosity of the logging output, from silent to trace. In some SDKs this is numeric, and in others string.
+
+The following example configures the Ably client to log only error messages (level 1) and processes them with the function @logWriteFun()@:
```

Comment:
It would be helpful to add a link to the numeric level translation here, so readers don’t have to look it up themselves

Reply:
Agreed, or just provide a small table here?